packages/cli/src/ceramic-daemon.ts

```diff
@@ -831,12 +834,12 @@ export class CeramicDaemon {
       return
     }
 
-    const withFieldIndices = req.query.withFieldIndices
-    if (withFieldIndices && withFieldIndices.toString().toLowerCase() == 'true') {
-      const indexedModels = await this.ceramic.admin.getIndexedModelsWithFieldIndices()
+    const withModelData = req.query.withModelData
```

Comment:
hmm... if we're talking about users explicitly opting-in to certain types of data to be returned or not, then I'd think being specific about what data they want would be better.  I.e. `withIndices` instead of `withModelData`.  The problem is that this arg is doing more than just determining whether or not index data is returned, its actually controlling the **format** of the output.

What about making this field `outputFormat` with two possible values (and defaulting to old behavior if the param is omitted)?  Maybe something like `document` and `streamidsOnly`.  Or `legacy` and `withMetadata`.  Or maybe call it `outputVersion` with values `0` or `1`?